src/Microsoft.Azure.WebJobs.Host/Bindings/FunctionBindingContext.cs

```diff
@@ -52,5 +60,10 @@ public TraceWriter Trace
         {
             get { return _trace; }
         }
+
+        /// <summary>
+        /// The short name of the current function. 
+        /// </summary>
+        public string MethodName { get; private set; }
```

Comment:
I'm just nervous about exposing the additional public surface area.  FuncDesc is non-public and has a poor shape. 

---
In reply to: [117082229](https://github.com/Azure/azure-webjobs-sdk/pull/1148#discussion_r117082229) [](ancestors = 117082229)